src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -59,6 +60,8 @@ public void init(FeedbackSessionAttributes feedbackSession, List<FeedbackQuestio
 
         buildPreviewForm(feedbackSession, studentList, instructorList);
 
+        this.shouldLoadInEditMode = shouldLoadInEditMode;
+
```

Comment:
@whipermr5 yes, the assignment was necessary. My bad, I thought got confused because of the little JSP trick with the getter methods. Have put back the changes. (Note, I had run god mode with these changes. Have also reverted that commit).